x/dualstaking/keeper/hooks.go

```diff
@@ -46,6 +46,10 @@ func (h Hooks) BeforeDelegationSharesModified(ctx sdk.Context, delAddr sdk.AccAd
 // create new delegation period record
 // add description
 func (h Hooks) AfterDelegationModified(ctx sdk.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) error {
+	if RedelegationFlag {
```

Comment:
can we call this flag with a flag that explains what it does, something like: disableDualstakingHook or something

Reply:
done